app/cdash/tests/CMakeLists.txt

```diff
@@ -599,5 +599,8 @@ set_tests_properties(/Feature/SubProjectDependencies PROPERTIES DEPENDS viewsubp
 add_php_test(misassignedconfigure)
 set_tests_properties(misassignedconfigure PROPERTIES DEPENDS /Feature/SubProjectDependencies)
 
+add_laravel_test(/Feature/RemoteWorkers)
+set_tests_properties(/Feature/RemoteWorkers PROPERTIES DEPENDS misassignedconfigure)
+
```

Comment:
Is there a more efficient place to put this in the dependency graph?  The only potential conflict I see comes from the file the test writes, so it seems like it would be straightforward to place in the graph such that it can be run in parallel with other tests.

Reply:
Good point. For now I'll remove this dependency altogether. We can add such a dependency later on if this proves to be an issue.